templates/index.html

```diff
@@ -1,11 +1,31 @@
-{% extends "base.html" %}
-{% block title %}Webeng Project{% endblock %}
-{% block link%}
-<link rel="stylesheet" href="{{url_for('static', filename='css/index.css')}}">
-{% endblock %}
-{% block script%}
-<script src="{{url_for('static', filename='js/index.js')}}"></script>
-{% endblock %}
-{% block content %}
-<h1>Hello World from project!</h1>
+{% extends "base.html" %}
+{% block title %}Webeng Project{% endblock %}
+{% block link%}
+<link rel="stylesheet" href="{{url_for('static', filename='css/index.css')}}">
+{% endblock %}
+{% block script%}
+<script src="{{url_for('static', filename='js/index.js')}}"></script>
+{% endblock %}
+{% block content %}
+
+<div class="d-flex mt-auto mb-auto">
+    <div class="container text-center w-50" style="border-radius: 10px">
+        <h1 class="mb-3">Kodenavne</h1>
+        <form class="text-dark" method="post" action="/room/create">
+            <div class="form-floating mb-3">
+                <input type="text" id="idUsernameInput" name="username" placeholder="Username" class="form-control"></input>
+                <label for="idUsernameInput">Username</label>
+            </div>
+            <div class="form-floating mb-3">
+                <select id="idLanguageSelect" class="form-select">
+                    <option selected value="en">English</option>
+                    <option value="de">Deutsch</option>
+                </select>
+                <label for="idLanguageSelect">Select your Language</label>
```

Comment:
Maybe use `Select game language` to clearly specify that it would be the in-game language
```suggestion
                <label for="idLanguageSelect">Select game language</label>
```